R-packages/covidcast/R/plot.R

```diff
@@ -346,32 +362,23 @@ plot_choro = function(x, time_value = NULL, include = c(), range,
       package = "covidcast"))
     map_df = map_df %>% dplyr::filter(map_df$LSAD == 'M1') # only get metro and not micropolitan areas
     if (length(include) > 0) {
-      map_df = map_df %>% dplyr::filter(map_df$GEOID %in% include)
+      map_df = map_df %>% dplyr::filter(
+        substr(.$NAME, nchar(.$NAME) - 1, nchar(.$NAME)) %in% include)
```

Comment:
What is this `substr()` call doing?

Reply:
It's getting the last two characters; the names are shown as 'Allegheny, PA' and the state is needed.

Reply:
Can you add this as a comment please?